crates/proto/src/dnssec/mod.rs

```diff
@@ -27,14 +26,13 @@ mod dnssec_dns_handle;
 #[doc(hidden)]
 pub use dnssec_dns_handle::verify_nsec;
 pub use dnssec_dns_handle::DnssecDnsHandle;
+/// ring implementations of DNSSEC traits.
```

Comment:
I'm guessing the point of renaming this is to decrease the specific reliance on `ring`, so this comment is probably wrong.

Reply:
Fair, fixed that up.